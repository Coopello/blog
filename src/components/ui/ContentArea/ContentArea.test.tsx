import renderer from "react-test-renderer";

import { ContentArea } from ".";

describe("ui/ContentArea", () => {
  it("Snap Shot", () => {
    const component = renderer.create(<ContentArea />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
