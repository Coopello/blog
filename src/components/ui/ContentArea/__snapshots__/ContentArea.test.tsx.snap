// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/ContentArea Snap Shot 1`] = `
<article
  className="prose max-w-none rounded bg-white p-8 lg:prose-base"
  id="test"
>
  <h1>
    test
  </h1>
</article>
`;
